vacasa/connect/connect.py

```diff
@@ -128,6 +128,65 @@ def get(self, uri, params: dict = None):
 
         return r.json()
 
+    """ Create a unit via connect. Required args are at the top of the list.
+        https://vacasa.docs.stoplight.io/units/postv1units
+    """
+    def create_unit(self,
+                    region_id: int,
+                    turnover_day: int,
+                    housing_type: str,
+                    code: str= None,
+                    name: str= None,
+                    bedrooms: int= None,
+                    full_baths: int= None,
+                    half_baths: int= None,
+                    max_occupancy: int= None,
+                    latitude: int= None,
+                    longitude: int= None,
+                    amenity_email: str= None,
+                    m_source: str= None,
```

Comment:
since the connect docs don't describe what some of these mean, it might be nice to have that in the docstring